python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -56,6 +56,7 @@ def __init__(
         **kwargs,
     ) -> None:
         kwargs.setdefault("task_id", get_unique_task_id("load_file"))
+        self.skip_core_execution = kwargs.pop("skip_core_execution", None)
```

Comment:
We'll also need to have this logic at least in `transform`/`transform_file` if we are to focus on the operators we currently support in the SQL CLI. We could probably also benefit by having it in `export_file`.